code/modules/mining/ore_redemption_machine/survey_vendor.dm

```diff
@@ -55,6 +56,7 @@
 		EQUIPMENT("Defense Equipment - Razor Drone Deployer",	/obj/item/weapon/grenade/spawnergrenade/manhacks/station/locked,	100),
 		EQUIPMENT("Defense Equipment - Sentry Drone Deployer",	/obj/item/weapon/grenade/spawnergrenade/ward,						150),
 		EQUIPMENT("Defense Equipment - Frontier Carbine",		/obj/item/weapon/gun/energy/locked/frontier/carbine,				750),
+		EQUIPMENT("Defense Equipment - Frontier Holdout Plus",	/obj/item/weapon/gun/energy/locked/frontier/holdout/holdoutplus,	500),
```

Comment:
I do not feel like there needs to be a Holdout plus, on top of that aside from stats it has no distinction from the regular holdout. This should be removed.

Reply:
it should  charge as fast as the carbine, if I haven't timed it wrong

Reply:
it's gone from the list